.github/workflows/get_modular_resources.py

```diff
@@ -9,12 +9,10 @@
 
 async def connect() -> ViamClient:
     dial_options = DialOptions(
-        # The URL of any robot in the location.
-        auth_entity='divine-voice-main.b5luvepsyg.viam.cloud',
+        auth_entity='744b0c18-f6fd-4c8d-a707-d3f261b353cc'
```

Comment:
should this still be `divine-voice-main.b5luvepsyg.viam.cloud`?

Reply:
The auth_entity is the api key id. I did test this, and it does work.

Reply:
Please do confirm I got this right for https://docs-test.viam.dev/1916/public/program/apis/data-client/ and https://docs-test.viam.dev/1916/public/program/apis/cloud/